### ISSUE TITLE
Issue with /gbblockinfo

### ISSUE THREAD

**Issue by rebel24** [GriefPrevention/GriefPrevention#324]
### What behaviour is observed:
_What you saw_ error in console
### What behaviour is expected:
_What you expected to see_ information on block
### Steps/models to reproduce: /gpblockinfo
_This may include a build schematic, a video, or detailed instructions to help reconstruct the issue_

### Spigot/Craftbukkit/Paper and GriefPrevention version: latest 1,13 and latest GP
___This can be found by running `/version` and `/version GriefPrevention` on your server.___

### Stack trace/error
```
[14:32:53 INFO]: rebel24 issued server command: /gpblockinfo
> [14:32:53 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'gpblockinfo' in plugin GriefPrevention v16.8
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at org.bukkit.craftbukkit.v1_13_R1.CraftServer.dispatchCommand(CraftServer.java:692) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.PlayerConnection.handleCommand(PlayerConnection.java:1634) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.PlayerConnection.a(PlayerConnection.java:1469) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.PlayerConnectionUtils.a(SourceFile:10) ~[server.jar:git-Spigot-0b44fa0-49a2604]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_171]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_171]
at net.minecraft.server.v1_13_R1.SystemUtils.a(SourceFile:198) [server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.MinecraftServer.w(MinecraftServer.java:884) [server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.DedicatedServer.w(DedicatedServer.java:411) [server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.MinecraftServer.v(MinecraftServer.java:819) [server.jar:git-Spigot-0b44fa0-49a2604]
at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:717) [server.jar:git-Spigot-0b44fa0-49a2604]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.util.IllegalFormatConversionException: d != org.bukkit.Material
at java.util.Formatter$FormatSpecifier.failConversion(Formatter.java:4302) ~[?:1.8.0_171]
at java.util.Formatter$FormatSpecifier.printInteger(Formatter.java:2793) ~[?:1.8.0_171]
at java.util.Formatter$FormatSpecifier.print(Formatter.java:2747) ~[?:1.8.0_171]
at java.util.Formatter.format(Formatter.java:2520) ~[?:1.8.0_171]
at java.util.Formatter.format(Formatter.java:2455) ~[?:1.8.0_171]
at java.lang.String.format(String.java:2940) ~[?:1.8.0_171]
at me.ryanhamshire.GriefPrevention.GriefPrevention.onCommand(GriefPrevention.java:2649) ~[?:?]
at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[server.jar:git-Spigot-0b44fa0-49a2604]
... 15 more
```
### GriefPrevention config.yml (if applicable)

```yml
Default config```

### Plugin list (if applicable):

**Comment by bigpresh** [comment]
The current released version of GP doesn't support 1.13 yet.

You didn't say what GP version you have, so I'm going to assume it's a released version and not one built from the 1.13 support PR that has only just been merged, but if you have built your own from that and are still getting this error, then please let us know.

**Comment by RoboMWM** [comment]
Duplicate of #313
